rhmap-mbaas-config.json

```diff
@@ -119,6 +119,12 @@
       "displayName": "Proxy Url",
       "value": "",
       "required": false
+    },
+    {
+      "name": "PROJECT_NAME",
+      "displayName": "Project name",
```

Comment:
Maybe change this to something more descriptive "RHMAP MBaaS project namespace" for example would be more intuitive. What do you think?